backend/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "target": "es6",
+    "module": "commonjs",
+    "outDir": "./dist",
```

Comment:
Well done getting the build to end up in the `dist` folder. I've definitely forgotten this step a few times in past

Reply:
Thank you, but tbh I referenced another tsconfig boilerplate. 
This option sets the target dir for TS builds, right?

Reply:
Yes when it builds your API it moves the compilated files into a `/dist` folder. If it was specified wrong you can potentially end up with all your TS files ending up in the same folders as your TS files :)